src/utils/daos_hdlr.c

```diff
@@ -1731,6 +1798,14 @@ fs_copy_dir(struct cmd_args_s *ap,
 				D_GOTO(out, rc);
 			(*num_files)++;
 			break;
+		case S_IFLNK:
+			rc = fs_copy_symlink(ap, src_file_dfs, dst_file_dfs,
+					     &next_src_stat, next_src_path,
+					     next_dst_path);
+			if (rc != 0)
+				D_GOTO(out, rc);
+			(*num_files)++;
```

Comment:
I think we probably need a `num_links` to count the number of links independently of files.